homeassistant/components/enocean.py

```diff
@@ -58,9 +58,10 @@ def send_command(self, command):
 
     def _combine_hex(self, data):  # pylint: disable=no-self-use
         """Combine list of integer values to one big integer."""
-        output = 0x00
+        output = 0
+        data = data.replace(' ', '').strip('[]').split(',')
```

Comment:
This assumes `data` is a string, while the old implementation assumed it was a list of integer values.